src/object/srv_obj.c

```diff
@@ -876,6 +876,22 @@ obj_singv_ec_rw_filter(struct obj_rw_in *orw, daos_iod_t *iods, uint64_t *offs,
 	}
 }
 
+/* Call internal method to increment CSUM media error. */
+static void
+obj_log_csum_err(void)
+{
+	struct dss_module_info	*info = dss_get_module_info();
+	struct bio_xs_context	*bxc  = info->dmi_nvme_ctxt;
+
```

Comment:
I've been seeing a lot of SDL patches lately checking NULL conditions.  do we need a D_ASSERT(info != NULL) here?

Reply:
I can add the assert.  The dss_module_info is initialized during module load.  I don't think we'd get this far in the server if it failed.  However, an assert failure is preferable to a seg fault.

Reply:
It's more for the security scans than whether it can happen or not